play/src/front/Components/MapEditor/PropertyEditor/PersonalAreaPropertyEditor.svelte

```diff
@@ -0,0 +1,130 @@
+<script lang="ts">
+    import { createEventDispatcher } from "svelte";
+    import { InfoIcon } from "svelte-feather-icons";
+    import { InputTagOption, PersonalAreaAccessClaimMode, PersonalAreaPropertyData } from "@workadventure/map-editor";
+    import LL from "../../../../i18n/i18n-svelte";
+    import InputTags from "../../Input/InputTags.svelte";
+    import MemberAutocomplete from "../../Input/MemberAutocomplete.svelte";
+    import PropertyEditorBase from "./PropertyEditorBase.svelte";
+
+    export let personalAreaPropertyData: PersonalAreaPropertyData;
+
+    let areaOwnerIdInput: string | undefined = personalAreaPropertyData.ownerId;
+    let _tags: InputTagOption[] | undefined = personalAreaPropertyData.allowedTags
+        ? personalAreaPropertyData.allowedTags.map((allowedTag) => ({
+              value: allowedTag,
+              created: false,
+              label: allowedTag,
+          }))
+        : undefined;
+
+    const dispatch = createEventDispatcher();
+
+    function setOwnerId(selectedOwnerId: string) {
+        personalAreaPropertyData.ownerId = selectedOwnerId;
+        areaOwnerIdInput = selectedOwnerId;
+        dispatch("change");
+    }
+
+    function handleTagChange(tags: InputTagOption[] | undefined) {
+        if (tags) {
+            personalAreaPropertyData.allowedTags = tags.map((tag) => tag.value);
```

Comment:
In the `InputTagsOption.ts` file, you could write a utility function get map to string[].

Something like:

```
export function toTags(InputTagOption[]): string[]
```